src/main/java/com/microsoft/sqlserver/jdbc/SQLServerConnection.java

```diff
@@ -132,7 +132,7 @@ public class SQLServerConnection implements ISQLServerConnection, java.io.Serial
     /**
      * Back off interval in ms for retries
      */
-    static final int BACKOFF_INTERVAL = 100;
+    static final int BACKOFF_INTERVAL = 500;
```

Comment:
this shouldn't exist we should be using the connectRetryInterval from connection string